### ISSUE TITLE
Changes in water demand

### ISSUE THREAD

**Issue by qianweihao** [kthorp/pyfao56#45]
Hello, I have a question, how do I determine the change in water requirements for different soil depths.For example, if I divide the root depth of the crop into 5 layers of soil and I get an ETc that should be the total water requirement for all 5 layers, how do I determine the water requirement for each layer? Is it as simple as excepting 5?

**Comment by kthorp** [comment]
FAO-56 is a relatively crude model and doesn't consider water balances in individual soil layers or redistribution of water between layers.  By default, FAO56 considers only a single homogeneous root zone (water holding parameters specified in the Parameters class). We have also implemented an optional approach in pyfao56 where a users can input soils data in different soil layers (SoilProfile class), and the model uses these data to compute total available water (TAW) over the entire root zone (with differential soil properties in different layers).  However, the model's water balance is still computed at the level of the entire root zone.  I think this makes your question somewhat moot.  As we have implemented it, the model cannot provide information about water requirements in different layers.  It can only give you information about depletion of the actively growing root zone (Dr) or depletion of the maximum root zone (Drmax).  Does that help?
